app/pb_kits/playbook/pb_date/_date.html.erb

```diff
@@ -1,14 +1,70 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname) do %>
-    <% if object.size == "lg" %>
-      <%= pb_rails("title", props: { text: object.lg_date, size: 3 }) %>
-    <% elsif object.size == "sm" %>
-      <%= pb_rails("icon", props: { icon: "calendar", fixed_width: true }) %>
-      <%= pb_rails("title", props: { tag: "span", text: object.sm_date, size: 4 }) %>
+    class: object.classname,
+    aria: object.aria) do %>
+
+  <% if object.size == "md" %>
```

Comment:
You mentioned that `size` will be deprecated and not used. Is this an exception to that?

Reply:
That's no longer the case since @kre8sions wants a small version of date for Pulse. However, I updated the date to follow the Time kit: Small and Medium.